packages/camera/camera/README.md

```diff
@@ -126,7 +127,16 @@ class _CameraAppState extends State<CameraApp> {
   @override
   void initState() {
     super.initState();
-    controller = CameraController(_cameras[0], ResolutionPreset.max);
+    controller = CameraController.withSettings(
+      _cameras[0],
+      const MediaSettings(
+        resolutionPreset: ResolutionPreset.low,
+        fps: 15,
+        videoBitrate: 200000,
+        audioBitrate: 32000,
+        enableAudio: true,
```

Comment:
Do we really want all of these parameters in the README example? I would not expect most users to need to control bitrates, and they certainly shouldn't need to set default audio settings.

Reply:
README, and example reverted to original.
But, I've been in the video processing industry for a long time, 100% of real customers request bitrate control.

Reply:
The issue this PR fixes has 68 votes across 4 years (not counting the time that `camera` existing without anyone requesting it). I assure you that `camera` has more than 68 real users.